sql_generators/active_users/templates/view.sql

```diff
@@ -3,7 +3,8 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ app_name }}.active_users_aggregates`
 AS
 SELECT
-  * EXCEPT (app_version),
+  * EXCEPT (app_version, app_name),
```

Comment:
We exclude `app_name` and then include it?

Reply:
This is to enable the UNIONED view of desktop + mobile.